modules/unMuteOverlayButton/resources/unMuteOverlayButton.js

```diff
@@ -56,12 +56,6 @@
                 this.getComponent().fadeOut('slow');
```

Comment:
To apply this to the hide function and keep the destroy function.
```suggestion
                hide: function () {
                this.getComponent().fadeOut('slow', function () {this.remove()});
            },

            destroy: function () {
                this.hide();
                this.unbind('playerReady');
                this.unbind('volumeChanged');
            },
```